sample/src/main/res/layout-sw600dp/app_bar_child_detail.xml

```diff
@@ -47,6 +47,15 @@
                     android:layout_weight="1"
                     android:orientation="vertical">
 
+                    <org.smartregister.view.customcontrols.CustomFontTextView
+                        android:id="@+id/outOfCatchment"
+                        style="@style/CustomFontTextViewStyle.ClientList.Medium"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="@string/out_of_catchment"
+                        android:textAppearance="@style/TextAppearance.AppCompat.Small"
+                        android:textColor="@android:color/holo_orange_light" />
```

Comment:
Consider a smaller font for *Out of Catchment* text to prevent view content overflowing, i.e., `Next Visit` date is down pushed beyond the view area